components/application/widgets/CheckBoxButton.js

```diff
@@ -16,7 +16,14 @@ import Icon from "./Icon";
 import { useDispatch } from "react-redux";
 import { completeTaskAction } from "../../../store/tasks";
 
-const CheckBoxButton = ({ priority, taskId, dueDate, projectId }) => {
+const CheckBoxButton = ({
+  priority,
+  taskId,
+  dueDate,
+  projectId,
+  completed,
+}) => {
+  const [isCompleted, setIsCompleted] = React.useState(completed);
```

Comment:
Why not import `react` and only use useState ?

Reply:
I didnt notice this. May this part is written by Copilot. Fixed